modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/fds_view.jsp

```diff
@@ -20,13 +20,13 @@ renderResponse.setTitle(ParamUtil.getString(request, "fdsViewLabel"));
 	module="{FDSView} from frontend-data-set-views-web"
 	props='<%=
 		HashMapBuilder.<String, Object>put(
+			"backURL", fdsViewsURL
```

Comment:
We might as well implement the standard back pattern, if we're relying on it DXP-wide with item actions :sweat_smile: